whitelist.txt

```diff
@@ -374,6 +374,7 @@ reentrant
 repo
 repo's
 repos
+returncode
 returncontract
```

Comment:
need to remove returncontract then.

Reply:
nope, it's in the changelogs and linter complained about it.

Reply:
We should move the previous releases changelogs to some other file to be able to clean up the white list, but that's for another PR for sure :)